model/actors/miner/sectorposts.go

```diff
@@ -0,0 +1,54 @@
+package miner
+
+import (
+	"context"
+
+	"github.com/go-pg/pg/v10"
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/label"
+	"golang.org/x/xerrors"
+)
+
+type MinerSectorPost struct {
```

Comment:
You'll want to add this to the list here: https://github.com/filecoin-project/sentinel-visor/blob/master/storage/sql.go#L29